### PR TITLE
remove cert-manager from serving install

### PR DESCRIPTION
**Pull request by RichieEscarez**
fixes: #1614 

the cert-manager controller is no longer installed by default

Related UX issue: https://github.com/knative/serving/issues/4120

### CODE REVIEW

**Comment by RichieEscarez** [comment]
Staged preview at: https://prstaging--knative-v1.netlify.com/docs/install/

Examples: 
https://prstaging--knative-v1.netlify.com/docs/install/knative-custom-install/#install-details-and-options
https://prstaging--knative-v1.netlify.com/docs/serving/using-auto-tls/#enabling-automatic-certificate-provisioning

File: docs/install/Knative-with-Ambassador.md [hunk 1]
```diff
@@ -158,5 +147,7 @@ manage and route to serverless applications!
 - Try the
   [Getting Started with App Deployment guide](./getting-started-knative-app/)
   for Knative serving.
-- Take a look at the rest of what
-  [Knative has to offer](https://knative.dev/docs/index.html)
+- Get started with Knative Eventing by walking through one of the
+  [Eventing Samples](../eventing/samples/).
+- [Install Cert-Manager](../serving/installing-cert-manager.md) if you want to use the
```
Comment (ZhiminXiang) [inline docs/install/Knative-with-Ambassador.md]: At this point, auto TLS only works with Istio. So I would not put auto TLS under Ambassador installation guide.

Comment (RichieEscarez) [reply docs/install/Knative-with-Ambassador.md]: Removed.

**Pull request merged**